### PR TITLE
Add tests for helper functions.

### PR DESCRIPTION
**Pull request by martiros**
Add tests for array_where, head, last, class_basename, str_limit, str_random, str_finish functions.

### CODE REVIEW

File: tests/Support/SupportHelpersTest.php [hunk 1]
```diff
@@ -169,6 +169,13 @@ public function testStrIs()
 		$this->assertFalse(str_is('foo?bar', 'fobar'));
 	}
 
+	public function testStrRandom()
+	{
+		$result = str_random(20);
+		$this->assertTrue(is_string($result));
+		$this->assertEquals(20, mb_strlen($result));
```
Comment (GrahamCampbell) [inline tests/Support/SupportHelpersTest.php]: Don't use the mb function.

**Comment by GrahamCampbell** [comment]
Please send to 5.0.

**Comment by martiros** [comment]
Done. https://github.com/laravel/framework/pull/7326

**Comment by GrahamCampbell** [comment]
You don't need to link me to them. I can see everything that goes on. :)

**Comment by martiros** [comment]
OK :)
I am new in github...

**Pull request closed without merging**